jsonschema-module-jackson/src/main/java/com/github/victools/jsonschema/module/jackson/JsonEnumDefaultValueResolver.java

```diff
@@ -0,0 +1,20 @@
+package com.github.victools.jsonschema.module.jackson;
+
+import com.fasterxml.jackson.annotation.JsonEnumDefaultValue;
+import com.github.victools.jsonschema.generator.TypeScope;
+import java.lang.reflect.Field;
+import java.util.Arrays;
+
+public class JsonEnumDefaultValueResolver {
+
+    public static String apply(TypeScope typeScope) {
+        if (typeScope.getType().getErasedType().isEnum()) {
+            return Arrays.stream(typeScope.getType().getErasedType().getDeclaredFields())
+                    .filter(enumValue -> enumValue.isAnnotationPresent(JsonEnumDefaultValue.class))
+                    .findFirst()
+                    .map(Field::getName)
```

Comment:
**suggestion (blocking)**: This is too simplistic.
There are several ways to represent an enum value in the JSON (Schema). The declared name in the code may therefore not be the appropriate one to declare as default here.
We should at least attempt to cover the handful of standard options already supported by the main schema generator and this `JacksonModule` (e.g., considering `@JsonValue` annotations).